frontend/src/app/globals.css

```
@@ -185,8 +242,8 @@
 }
 
 @theme inline {
-  --font-sans: var(--font-display-sans), "Plus Jakarta Sans", sans-serif;
-  --font-mono: var(--font-geist-mono), "Geist Mono", monospace;
+  --font-sans: var(--font-sans-loaded), var(--font-display-sans), "Plus Jakarta Sans", system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)